README.md

```diff
@@ -416,8 +416,8 @@ where:
 * `conditions` (`?table`) - array of [select conditions](#select-conditions)
 * `opts`:
   * `first` (`?number`) - the maximum count of the objects to return.
-     If negative value is specified, the last objects are returned
-     (`after` option is required in this case).
+     If negative value is specified, the objects behind `after` are returned
+     (`after` option is required in this case). [See pagination examples](https://github.com/tarantool/crud/blob/master/doc/select.md#pagination).
```

Comment:
Nit: We can use a relative link here: `[pagination examples](doc/select.md#pagination)`.

Reply:
Wow, It works. Thanks!